code/game/objects/items/weapons/storage/pouches.dm

```diff
@@ -162,6 +162,25 @@
 		/obj/item/gun/projectile/flare_gun
 		)
 
+/obj/item/storage/pouch/engineering_material
+	name = "engineering material pouch"
+	desc = "Can hold sheets, rods and cable coil. Has only 2 slots."
```

Comment:
You should bring this in line with the other descriptions, "has only 2 slots" isn't fit.

Reply:
done, removed the "has only 2 slots". Couldnt think of anything else so its just "Can hold sheets, rods and cable coil"